.env.tpl

```diff
@@ -38,3 +38,5 @@ SATNAV_BUCKET_NAME = ''
 MAILSLURP_API_KEY = ''
 MAILSLURP_TIMEOUT = '120000'
 
+# Feature flags
+REQUIRE_PAYMENT_PLAN = 'true'
```

Comment:
would be good to test this in staging. I am not 100% sure if swapping env var is enough, or we need to re-deploy. So knowing that can help us know what we need to do